styles/styles.css

```diff
@@ -375,14 +400,185 @@ body {
   max-height: 295px;
 }
 
-.homes__name {
+.places__name {
   color: #3077c6;
+  text-decoration: none;
 }
 
 .homes__location {
   color: #bfbfbf;
 }
 
+.destinations {
+  background-color: #ffffff;
+}
+
+.destinations__container {
+  padding-bottom: 210px;
+}
+
+.destinations__menu {
+  display: flex;
+  background-color: #f3f3f4;
+  border-radius: 8px;
+  margin-bottom: 65px;
+}
+
+.destinations__menu-item {
+  display: flex;
+}
+
+.destinations__menu-name {
+  text-decoration: none;
+  padding-top: 16px;
+  padding-bottom: 16px;
+  background-color: #f3f3f4;
+  border-radius: 8px;
+  width: 100%;
+  text-align: center;
+  transition: background-color 300ms ease;
+}
+
+.destinations__menu-item:hover a {
+  background-color: #3077c6;
+  color: #ffffff;
+}
+
+.destinations__img-block {
+  position: relative;
+}
+
+.overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(56, 56, 56, 0);
+  transition: background-color 0.5s ease;
+}
+
+.destinations__img-block:hover .overlay {
+  background-color: rgba(56, 56, 56, 0.4);
+}
+
+.destinations__booking {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  text-align: center;
+  opacity: 0;
+  transition: opacity 0.5s ease;
+}
+
+.destinations__booking-submit {
```

Comment:
Yoy write good code. But for make it more readable it will be great if you rearrange properties in classes (start it with position, display, when sizes and gaps and at last other decorations)